src/NServiceBus.Core/Sagas/ActiveSagaInstance.cs

```diff
@@ -170,7 +165,18 @@ static object GetDefault(Type type)
             return null;
         }
 
-        Dictionary<string, object> correlatedPropertyValues = new Dictionary<string, object>();
+        internal IDictionary<string, object> CorrelationProperties
+        {
+            get { return correlationProperties.ToDictionary(cp => cp.PropertyInfo.Name, cp => cp.Value); }
```

Comment:
Ok

On Tuesday, October 27, 2015, Andreas Öhlund <notifications@github.com>
wrote:

> In src/NServiceBus.Core/Sagas/ActiveSagaInstance.cs
> <https://github.com/Particular/NServiceBus/pull/3012#discussion_r43101050>
> :
>
> > @@ -170,7 +165,18 @@ static object GetDefault(Type type)
> >              return null;
> >          }
> >
> > -        Dictionary<string, object> correlatedPropertyValues = new Dictionary<string, object>();
> > +        internal IDictionary<string, object> CorrelationProperties
> > +        {
> > +            get { return correlationProperties.ToDictionary(cp => cp.PropertyInfo.Name, cp => cp.Value); }
>
> I'm about to change the api to only pass a single prop in a different pull
> so given that this is only called once I propose we leave it as is for now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3012/files#r43101050>.
>


-- 
*#NSBcon 2015 • The "All About NServiceBus" conference is back *
*• Learn more <http://go.particular.net/l/27442/2015-08-21/5d4wzp> →*